eng/mgmt/mgmtmetadata/app_resource-manager.txt

```diff
@@ -0,0 +1,15 @@
+﻿Installing AutoRest version: v2
+AutoRest installed successfully.
+Commencing code generation
+Generating CSharp code
+Executing AutoRest command
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/main/specification/app/resource-manager/readme.md --csharp --version=2.0.4421 --reflect-api-versions --csharp-sdks-folder=D:\DotNet-SDK\May-2022\azure-sdk-for-net\sdk
```

Comment:
All Track 1 SDKs will be deprecated soon As the Track 2 SDK has released stable (GA) version. So from now any Track 1 SDK release PR will now be approved and release to Nuget, please contact us if you still need a Track 1 release and ask for an exemption.
And here for ContainerApps there already have a beta version https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/containerapps/Azure.ResourceManager.Applications.Containers

Reply:
@ArthurMa1978 we want to release this for //BUILD. Is the beta version of track2 you linked using the latest swagger changes & stable version. if not can we get this released for //BUILD & we will work on updating the Track2 version  as well?

Reply:
@panchagnula , @ArthurMa1978 - the pre-release does not look up-to-date. The parameter names do not match the azure-rest-api-spec. For example:

Method to deactivate revision uses "name" as the revision name parameter: https://github.com/Azure/azure-sdk-for-net/blob/3046996080441d92a3137042add76b3d8dcc5759/sdk/containerapps/Azure.ResourceManager.Applications.Containers/src/Generated/RestOperations/ContainerAppsRevisionsRestOperations.cs#L345

However the spec uses "revisionName": https://github.com/Azure/azure-rest-api-specs/blob/b9d9355460992ba575206a96db8556985f579d9d/specification/app/resource-manager/Microsoft.App/stable/2022-03-01/ContainerAppsRevisions.json#L178